packages/flutter_tools/templates/app/linux.tmpl/app.desktop.tmpl

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name={{projectName}}
+Comment={{description}}
+Icon=icon.png
+Exec={{projectName}} %U
```

Comment:
This should be the binary name, set from the install script, so it updates correctly if someone renames the binary.